NEWS.md

```diff
@@ -5,6 +5,7 @@
 ### Bug fixes
 
 * `$len()` now correctly includes `null` values in the count (#1044).
+* cache temporary file name for url file download (#1046)
```

Comment:
```suggestion
* In some read/scan functions on Windows, downloading files could fail if the URL was too long. This is now fixed (#1049).
```
The NEWS item should mention the problem solved rather than the solution employed. I took the liberty of adding the OS because I think this is a windows-specific issue but you can remove that if the problem occurred on another OS for you.

Reply:
Thanks for the info, I edited the title and the suggested change

Reply:
Ah my bad for being too eager and not refreshing my browser 😆